src/common/transformations/src/transformations/common_optimizations/pack_GQA.cpp

```diff
@@ -0,0 +1,310 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "transformations/common_optimizations/pack_GQA.hpp"
+
+#include "openvino/core/graph_util.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/matmul.hpp"
+#include "openvino/op/reduce_sum.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/transpose.hpp"
+#include "openvino/pass/pattern/op/optional.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/utils/block_collection.hpp"
+#include "transformations/utils/utils.hpp"
+
+using namespace ov;
+using namespace ov::op;
+using namespace ov::pass;
+using namespace ov::pass::pattern;
```

Comment:
```suggestion
namespace ov::pass {
```
by default wrap the implementation in same namespace as in header add using for what is required.